analyze.go

```diff
@@ -250,6 +250,12 @@ outer:
 			a = appendSplit(a, splitBy, v)
 		}
 	}
+
+	if defaultValue, is := v.Default(); is {
+		if o, _ := v.Expr(); o == cue.NoOp {
+			a = append(a, defaultValue)
+		}
+	}
```

Comment:
this feels like it could have significant and unpredictable effects - have you run this against the kinds in grafana and ensured no output changes?

Reply:
> have you run this against the kinds in grafana and ensured no output changes?

Isn't that an implicit suggestion to grow the `testdata` base? 🤔

Reply:
i mean...you're not wrong :)

it'd be great to have all that programmatically available as a testing corpus